Entities/MOSRotating.cpp

```diff
@@ -1416,7 +1417,7 @@ void MOSRotating::Update() {
         m_OrientToVel = std::clamp(m_OrientToVel, 0.0F, 1.0F);
 
         float velInfluence = std::clamp(m_OrientToVel < 1.0F ? m_Vel.GetMagnitude() / 100.0F : 1.0F, 0.0F, 1.0F);
-        float radsToGo = m_Rotation.GetRadAngleTo(m_Vel.GetAbsRadAngle());
+        float radsToGo = m_Rotation.GetRadAngleTo(m_Vel.GetAbsRadAngle() + (m_HFlipped ? c_PI * GetFlipFactor() : 0));
```

Comment:
I have no recollection what I was thinking here but I assure you there was a reason behind this. I will, however, clean it up in case I can't remember that reason lol.

Reply:
Haha sounds good. If you remember the reason, do share :)